src/main/java/dev/aziz/movies/exceptions/ObjectNotValidException.java

```diff
@@ -0,0 +1,13 @@
+package dev.aziz.movies.exceptions;
+
+import lombok.Data;
+
+import java.util.Set;
+
+@Data
+public class ObjectNotValidException extends RuntimeException {
```

Comment:
Adding the `@Valid` annotation in the controller next to the RequestBody already checks all the constraints